src/components/Card/Card.tsx

```diff
@@ -63,6 +64,7 @@ export default class Card extends React.Component<CardProps, {
     if (props.type === "snippet") cardId = props.item.title.replaceAll(" ", "-");
     else if (props.type === "theme") cardId = props.item.manifest?.usercss || "";
     else if (props.type === "extension") cardId = props.item.manifest?.main || "";
+    else if (props.type === "app") cardId = props.item.manifest?.name?.replaceAll(" ", "-") || "";
```

Comment:
Why are we replacing spaces with hyphens here but no where else?

Reply:
It is only meant for localStorage as a means to not include spaces in keys

![image](https://user-images.githubusercontent.com/77577746/174482749-274c24fe-37e5-4eea-b589-f5f74fbc2024.png)